### ISSUE TITLE
prompt adds a new line after initialization

### ISSUE THREAD

**Issue by musm** [nickcox/pure-pwsh#8]
When starting powershell or pwsh with this prompt it ends up adding a new line at initialization.

I.e.  
```
$ pwsh -NoLogo

<pure prompt> # additional new line is inserted above
```

```
$ pwsh -NoLogo
<normal prompt> # no new line above
```

**Comment by nickcox** [comment]
This is as per the [pure behaviour,](https://github.com/sindresorhus/pure/issues/233) I think.

Does this only bother you at startup or would you prefer never to see a newline before the prompt?

**Comment by musm** [comment]
Yes it's very annoying.  Especially when pwsh is used in other applications such as  vscode.  Thanks for the prompt responses @nickcox

**Comment by musm** [comment]
As a short term solution I just removed  ```  `n ``` in https://github.com/nickcox/pure-pwsh/blob/master/pure-pwsh/prompt.ps1#L31

**Comment by nickcox** [comment]
Yeah, I think the newlines should be part of `$pure.prePrompt` to make it easier to customise.

**Comment by nickcox** [comment]
This is published as [0.6.0](https://www.powershellgallery.com/packages/pure-pwsh/0.6.0). You can remove either of the newlines from the default PrePrompt of:

```
{param ($cwd, $git, $slow) "`n$cwd $git $slow"`n}
```
I think special-casing the initialisation of a new prompt is probably not worth it though.

**Comment by musm** [comment]
Works well, thanks
